src/app/asset-page/asset-page.component.pug

```diff
@@ -196,9 +196,9 @@
                 //- Collections
                 ng-container(*ngIf="fieldName === 'Collection' && i === 0")
                   div(*ngFor="let col of assets[0].collectionLinks")
-                    a.value.col-link(id="collection-link", (click)="trackCollectionLink(cleanFieldValue(col.displayName))", [routerLink]="col.route", [innerHTML]="cleanFieldValue(col.displayName) | linkify", [attr.aria-labelledby]="Collections")
+                    a.value.meta-link.col-link(id="collection-link", (click)="trackCollectionLink(cleanFieldValue(col.displayName))", [routerLink]="col.route", [innerHTML]="cleanFieldValue(col.displayName) | linkify", [attr.aria-labelledby]="Collections")
```

Comment:
do we still need `meta-link` here?

Reply:
It wouldn't currently do anything, but it would provide style consistency if some style was applied to all meta-links?

Reply:
I might be overthinking it

Reply:
I am cool with leaving it in. Doesn't seem to be hurting anything.